about.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>qq puzzles</title>
+	<meta name="robots" content ="noindex,nofollow" />
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8" />
+	<link rel="stylesheet" href="css/bricks.css">
+	<link rel="stylesheet" href="css/styles.css">
+	<link rel="stylesheet" href="css/nav.css">
+</head>
+<body>
+	<div class='wrapper'>
+		<header>
+			<nav id='cssmenu'>
+				<ul>
+					<li><a href='index.html'>Home</a></li>
```

Comment:
it looks like you've got a reusable component here!

Unfortunately I know of no way of reusing pieces of HTML like this without slowly losing your mind in the jungle of frontend "Components" frameworks. "Vue" is the most in-vogue one at the moment. Others include "React" and "Angular".

I would also check out "Web Components", which is like a browser-native substrate for things like Vue, except the ergonomics are muuuuuuch worse. But probably good enough for this use case here. https://www.webcomponents.org/introduction